src/main/java/net/ripe/rpki/rsyncit/rrdp/RrdpFetcher.java

```diff
@@ -223,8 +227,9 @@ private ProcessPublishElementResult processPublishElements(Element doc, Instant
             .mapToObj(publishedObjects::item)
             .toList();
 
-        var t = Time.timed(() -> objectItems
-            .parallelStream()
+        var objects = metrics.objectConstructionTimer.record(() -> objectItems
+            .stream().unordered()
+            .parallel()
```

Comment:
Why add `unordered`?

Reply:
Because I do not think we need ordering. Will remove this because it's confusing.